ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml

```diff
@@ -1689,11 +1689,19 @@ presubmits:
         - ci-operator
         env:
         - name: CLUSTER_TYPE
-          value: openstack
+          value: openstack-vexxhost
         - name: JOB_NAME_SAFE
           value: e2e-openstack
         - name: KURYR_ENABLED
           value: "false"
+        - name: OPENSTACK_EXTERNAL_NETWORK
+          value: public
+        - name: OPENSTACK_FLAVOR
+          value: v1-standard-4
+        - name: OPENSTACK_MASTER_VOLUME_TYPE
+          value: rbd
```

Comment:
Why is the master volume back?

Reply:
Because https://github.com/openshift/release/pull/10594 hasn't merged yet and as a result we need to overwrite the `OPENSTACK_MASTER_VOLUME_TYPE` variables with values matching vexxhost.

Reply:
#10594 has landed, we don't need to set `OPENSTACK_MASTER_VOLUME_TYPE` anymore. Could you please update the PR?

Reply:
#10594 has now landed, could you update the PR to remove `OPENSTACK_MASTER_VOLUME_TYPE`?